### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by FraserSmith**
Updated with grammar corrections

### CODE REVIEW

File: Getting-Started/Data/Members/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Members
-*Members are used for registering and authenticating external users of an Umbraco installation (ie. forum members, intranet users and so forth). Unlike with Document Types and Media Types everything is done in the Members section both defining and creating, and editing members. This guide will explain how to define and create members in the backoffice. If you want to work with members using the service APIs, links can be found at the end of document*
+*Members are used for registering and authenticating external users of an Umbraco installation (ie. forum members, intranet users and so forth). Unlike with Document Types and Media Types everything is done in the Members section both defining and creating, and editing members. This guide will explain how to define and create members in the back office. If you want to work with members using the service APIs, links can be found at the end of the document*
```
Comment (jmayntzhusen) [inline Getting-Started/Data/Members/index.md]: 'backoffice' is the official word for it in Umbraco terminology, so it should be in one word 😉

File: Getting-Started/Data/Members/index.md [hunk 2]
```diff
@@ -90,13 +90,13 @@ __NOT A MEMBER OF GROUP(S):__ Lists all the groups that the member is not a part
 __MEMBER OF GROUP(S):__ Lists all the groups that the member is a part of. To remove the member from a group simply click it and it will move to the other column.
 
 # Sensitive data
-Umbraco 7.9.0 provides the ability to mark members as sensitive. A sensitive member's data will not be displayed to backoffice users unless they have appropriate permissions.
+Umbraco 7.9.0 provides the ability to mark members as sensitive. A sensitive member's data will not be displayed to back office users unless they have appropriate permissions.
```
Comment (jmayntzhusen) [inline Getting-Started/Data/Members/index.md]: Same as above, should be one word 🙂

File: Getting-Started/Data/Members/index.md [hunk 3]
```diff
@@ -90,13 +90,13 @@ __NOT A MEMBER OF GROUP(S):__ Lists all the groups that the member is not a part
...
 
 More information can be found under [security](../../../Reference/Security/#sensitive-data).
 
 # Technical
 As a developer you are able to leverage your website easily when you build on the Members section of Umbraco.
-Although the Members section is by default in the Umbraco backoffice, you will be able to implement some work on the front end of your website.
+Although the Members section is by default in the Umbraco back office, you will be able to implement some work on the front end of your website.
```
Comment (jmayntzhusen) [inline Getting-Started/Data/Members/index.md]: Should be one word again 😁

File: Getting-Started/Data/Members/index.md (hunk 1 above)
Comment (FraserSmith) [reply Getting-Started/Data/Members/index.md]: Ah, thank you for the heads up!
I have corrected this is my master - is this sufficient for the PR to be merged?
Thanks again 🙂

**Comment by FraserSmith** [comment]
Master updated based on changes requested :thumbsup:

File: Getting-Started/Data/Members/index.md (hunk 2 above)
Comment (FraserSmith) [reply Getting-Started/Data/Members/index.md]: :thumbsup:

File: Getting-Started/Data/Members/index.md (hunk 3 above)
Comment (FraserSmith) [reply Getting-Started/Data/Members/index.md]: :thumbsup:

**Comment by jmayntzhusen** [comment]
Hey @FraserSmith 

Yep this is great, thanks for the update! 😁

Do you have a profile on https://our.umbraco.com? we give all contributors a badge there!

**Comment by FraserSmith** [comment]
Ace, thank you very much for your help!

I do now, I am /member/275763 😄

**Comment by nul800sebastiaan** [comment]
Hi there @FraserSmith,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**